### DISCUSSION TITLE
Is Pluto compatible with JuliaPro 1.5?

### DISCUSSION THREAD

**Discussion by oviedorodolfo** [fonsp/Pluto.jl#1022] [Questions about using Pluto] [answered]
Hi,
I have JuliaPro 1.5.4-1 installed. Can I install Pluto from Juno's REPL?
Is there any advantage for doing otherwise?
Many thanks,
Rodolfo

**Comment by fonsp** [comment, marked as answer]
I don't really know, but you get the best Pluto experience on normal Julia. We recommend students taking https://computationalthinking.mit.edu/ to not install JuliaPro.
